### PR TITLE
Javascript implementation of client, part 3

### PR DESCRIPTION
**Pull request by baloyan**
Before reviewing current PR please make sure that #78 PR was reviewed and merged, if not, please review #78 PR.

- Add StringToarray() function, which converts string (integer numbers) into the byte array.
- Add TicketsBundle, BatchSignature, HashEncryptPuzzleInfo, TicketLotteryOne and TicketRequest objects' definition and implementation.

### CODE REVIEW

File: javascript_client/constants.js [hunk 1]
```diff
@@ -13,7 +13,9 @@
 var ECDSA_COMPRESSED_PUBLIC_KEY_SIZE = 33;
 var CONTENT_REQUEST_TRIALS = 3;
 var MAX_NUM_BLOCKS_PER_REQ = 5;
-
+var SHA256_HASH_SIZE = 32;
```
Comment (SantiagoTorres) [inline javascript_client/constants.js]: I'd imagine you want to use `const` here.

Comment (baloyan) [inline javascript_client/constants.js]: Yes you are right, to be honest just yesterday I found that JS can have constant variable.

Comment (SantiagoTorres) [inline javascript_client/constants.js]: Lol I get you, I'm also pretty new to the lang.

Comment (baloyan) [inline javascript_client/constants.js]: @SantiagoTorres is it good practice in JS also use `const` in function arguments ?

Comment (baloyan) [inline javascript_client/constants.js]: OK, I have never written in JS before, this is my first project in JS

**Comment by baloyan** [comment]
Please review #83 which is the latest one.

**Pull request closed without merging**